sky/unit/test/widget/input_test.dart

```diff
@@ -57,4 +58,43 @@ void main() {
     // Check that the Input with text has the same size as the empty Input.
     expect((input.renderObject as RenderBox).size, equals(emptyInputSize));
   });
+
+  test('Cursor blinks', () {
+    WidgetTester tester = new WidgetTester();
+
+    GlobalKey inputKey = new GlobalKey();
+
+    Widget builder() {
+      return new Center(
+        child: new Input(
+          key: inputKey,
+          placeholder: 'Placeholder'
+        )
+      );
+    }
+
+    new FakeAsync().run((async) {
+      tester.pumpFrame(builder);
+
+      EditableText editableText = tester.findWidget(
+          (Widget widget) => widget is EditableText);
+      Duration blinkDuration = new Duration(
+          milliseconds: editableText.test_cursorBlinkPeriodMs);
```

Comment:
Why not have the editableText expose this value as a Duration?  We've been trying to move more things to be expressed in Durations.